pandas/tests/series/test_ufunc.py

```diff
@@ -310,3 +310,18 @@ def __repr__(self):
     result = np.add(s, Thing(1))
     expected = pd.Series([Thing(2), Thing(3)])
     tm.assert_series_equal(result, expected)
+
+
+def test_outer():
+    # https://github.com/pandas-dev/pandas/issues/27186
+    s = pd.Series([1, 2, 3])
+    o = np.array([1, 2, 3])
+
+    with tm.assert_produces_warning(FutureWarning):
```

Comment:
what about np.outer? any other methods?

Reply:
np.outer isn't a ufunc, so it wouldn't hit here.

Haven't looked closely at the other methods yet.